build.gradle.kts

```
@@ -1,15 +1,15 @@
 plugins {
     java
-    id("fabric-loom") version "1.0.+"
+    id("fabric-loom") version "1.0.12"
```

```suggestion
    id("fabric-loom") version "1.0.+"
```

Fabric usually recommends using a dynamic version here